cloud-formation/membership-app.cf.yaml

```diff
@@ -87,6 +101,10 @@ Resources:
         Value:
           Ref: Stage
         PropagateAtLaunch: 'true'
+      - Key: Name
+        Value:
+          Fn::FindInMap: [ StageVariables, { Ref: Stage }, InstanceName ]
+        PropagateAtLaunch: 'true'
```

Comment:
this is just so the instances get a name so we can more easily tell which is which